app/customer.py

```diff
@@ -0,0 +1,25 @@
+from app.json_reader import data
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: List[int]
+    money: int
+    car: dict
+
+    @staticmethod
+    def create_customers() -> list:

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]